Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -6171,39 +6210,7 @@ - (void)_updateTextInputTraits:(id <UITextInputTraits>)traits
 #endif
 
     auto privateTraits = (id <UITextInputTraits_Private>)traits;
-    if ([privateTraits respondsToSelector:@selector(setIsSingleLineDocument:)]) {
-        switch (_focusedElementInformation.elementType) {
-        case WebKit::InputType::ContentEditable:
-        case WebKit::InputType::TextArea:
-            privateTraits.isSingleLineDocument = NO;
-            break;
-#if ENABLE(INPUT_TYPE_COLOR)
-        case WebKit::InputType::Color:
-#endif
-        case WebKit::InputType::Date:
-        case WebKit::InputType::DateTimeLocal:
-        case WebKit::InputType::Drawing:
-        case WebKit::InputType::Email:
-        case WebKit::InputType::Month:
-        case WebKit::InputType::Number:
-        case WebKit::InputType::NumberPad:
-        case WebKit::InputType::Password:
-        case WebKit::InputType::Phone:
-        case WebKit::InputType::Search:
-        case WebKit::InputType::Select:
-        case WebKit::InputType::Text:
-        case WebKit::InputType::Time:
-        case WebKit::InputType::URL:
-        case WebKit::InputType::Week:
-            privateTraits.isSingleLineDocument = YES;
-            break;
-        case WebKit::InputType::None:
-            break;
-        }
-    }
-
-    if ([privateTraits respondsToSelector:@selector(setShortcutConversionType:)])
-        privateTraits.shortcutConversionType = _focusedElementInformation.elementType == WebKit::InputType::Password ? UITextShortcutConversionTypeNo : UITextShortcutConversionTypeDefault;
+    [self _updatePrivateTextInputTraits:privateTraits];
```

Comment:
Hm... it's actually not super clear to me why this needs to move. Can we just add something like `_updateAdditionalTextInputTraits:`? (which is stubbed out for non-internal SDK builds).